backend-zs/models/story.model.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require('mongoose')
+
+var Story = mongoose.Schema({
+    title: {
+        type: String,
+        required: true,
+    },
+    author: {
+        type: String,
+        required: true,
+    },
+    authorImage: {
+        type: String,
+        required: false,
+    },
+    shortDescription: {
+        type: String,
+        required: true
+    },
+    story: {
+        type: String,
+        required: true,
+    },
+    category: {
+        type: String,
+        required: false,
+    },
+    numberLikes: {
+        type: [mongoose.Schema.Types.ObjectId],
+        ref: 'User',
+        default: []
+    },
+    games: [String]
+
+})
+
+var Story = mongoose.model('Story', Story)
```

Comment:
no need for the variable reuse here

```suggestion
module.exports = mongoose.model('Project', Project);
```

and it can then be imported from other classes as follows:
```js
var Project = require('../models/project.model');
```